cookiecutter/README.md

```diff
@@ -0,0 +1,18 @@
+# Streamlit template for CookieCutter
+
+Template for creating a streamlit component
+
+## Usage
+
+Make sure you have [https://pypi.org/project/cookiecutter/](cookiecutter) installed. To install it, you can use [pipx](https://pypa.github.io/pipx/):
+```sh
+pipx install cookiecutter
+```
+
+Then run
+// TODO: Remove --checkout cookieuctter before merge
```

Comment:
TODO

Reply:
What's TODO here? Can you describe it? Do we have a jira task for the follow-up?

Reply:
I saved more details in a file so that it could be copied more easily if someone wanted to use this command. I left a comment on GitHub to remember to delete it before merging.